tests/core/test_scripts.py

```diff
@@ -2657,17 +2675,19 @@ def test_do_run(self, db: DatabaseTransactionFixture):
         assert kwargs["subject"].__contains__("Inventory Report")
         attachments: dict = kwargs["attachments"]
 
-        assert len(attachments) == 1
-        key = [*attachments.keys()][0]
-        assert "biblioboard" in key
-        value = attachments[key]
-        assert len(value) > 0
-        csv_file = StringIO(value)
-        reader = csv.reader(csv_file, delimiter=",")
+        assert len(attachments) == 2
```

Comment:
Why two attachments, aren't we supposed to be putting all the files into a single attachment?